src/Bridge/Upscale/Blackfire/WithProfiler.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace K911\Swoole\Bridge\Upscale\Blackfire;
+
+use K911\Swoole\Server\Configurator\ConfiguratorInterface;
+use Swoole\Http\Server;
+use Upscale\Swoole\Blackfire\Profiler;
+
+final class WithProfiler implements ConfiguratorInterface
+{
+    /**
+     * @var Profiler
+     */
+    private $profiler;
+
+    public function __construct()
```

Comment:
```suggestion
    public function __construct(Profiler $profiler)
```

Then, you could write unit tests with the help of prohpecy

Reply:
Thanks, I'll look into it